src/app/views/cheat-sheets/mining/mining.component.html

```diff
@@ -6,6 +6,7 @@
                 <li>Ores have a <i>mining time</i>, and miners have <i>mining speed</i>.</li>
                 <li>Luckily, many of ores’ values are the same, so things aren’t as complicated as they could be.</li>
                 <li>See various <a href="https://imgur.com/a/Imfhz" target="_blank" rel="noopener">mining arrangements</a> with varying mining coverage.</li>
+                <li><a href="https://wiki.factorio.com/Mining_productivity_(research)" target="_blank" rel="noopener">Mining productivity</a> research increases the output of all mining drills and pumpjacks. <span class="text-muted small">Refer to an <a href="https://kirkmcdonald.github.io/calc.html#data=0-17-1&mprod=10&items=iron-ore:f:1" target="_blank" rel="noopener">online calculator</a> for more information (note mining productivity bonus on settings page)</span></li>
```

Comment:
* Instead of one long line please separate into multiple lines for readability.
* For the calc link, i think we should base it on belt speed, for example yellow belt is 15i/s
* https://kirkmcdonald.github.io/calc.html#data=0-17-1&rate=s&mprod=10&items=iron-ore:r:15